lib/aggregators/Max.ts

```diff
@@ -7,10 +7,7 @@ interface IExtremeState {
 export class Max extends BaseAggregator<IExtremeState> {
   public init(start: RDF.Term): IExtremeState {
     const { value } = this.extractValue(start);
-    if (start.termType === 'Literal') {
```

Comment:
`extractValue` already ensures that `start` is a literal.

Reply:
There might be some TS magic we can apply here to automatically let `start` be inferred as Literal.
Something like `start a RDF.Iteral` might be appendable to the return type of `extractValue` (but I don't see this as in-scope for this bounty, just optional :-) ).

Reply:
Also sounds good to me :-)